docs/index.md

```diff
@@ -15,8 +15,9 @@ offline analytics, background syncing, and more.
 
 ## A service worker in two minutes
 
-Use [sw-cli](sw-cli)
-to add a service worker to an existing project in under two minutes.
+Use [sw-cli](sw-cli) to add a service worker to an existing project in under two
+minutes. Save your cli responses to a configuration file so that updates are
```

Comment:
Just my two cents - this config file bit feels a bit too much info for the developer.